bedrock/products/templates/products/vpn/platforms/v2/linux.html

```diff
@@ -44,7 +44,9 @@ <h3>{{ ftl('vpn-shared-platform-what-is') }}</h3>
 <p>{{ ftl('vpn-linux-mozilla-vpn-is-a') }}</p>
 <h3>{{ ftl('vpn-linux-why-choose') }}</h3>
 <ul class="mzp-u-list-styled">
-  <li>{{ ftl('vpn-linux-fast-and') }}</li>
+  <li>
+    {{ ftl('vpn-linux-fast-and-v2', fallback='vpn-linux-fast-and') }}
```

Comment:
Thanks, I didn't know about the shorthand (took the first piece of code using a -v2 string).

Should I put these on the same line as the markup?

```
<li>{{ ftl('vpn-linux-fast-and-v2', fallback='vpn-linux-fast-and') }}</li>
```

Reply:
Yeah that's fine. We tend to indent for string length, but these are pretty short lines